test/proxyOFT.ts

```diff
@@ -0,0 +1,724 @@
+      "removeTrustedRemote(uint16)",
+    ];
+    const removeArray = new Array(functionregistry.length).fill(false);
+    await bridgeAdminRemote.upsertSignature(functionregistry, removeArray);
+    await bridgeAdminLocal.upsertSignature(functionregistry, removeArray);
+
+    // Setting local chain
+    await bridgeAdminLocal.setTrustedRemoteAddress(remoteChainId, remoteOFT.address);
+
+    let data = localOFT.interface.encodeFunctionData("setMinDstGas", [remoteChainId, 0, 200000]);
+    await deployer.sendTransaction({
+      to: bridgeAdminLocal.address,
+      data: data,
+    });
+    data = localOFT.interface.encodeFunctionData("setMaxDailyLimit", [remoteChainId, maxDailyTransactionLimit]);
+    await deployer.sendTransaction({
+      to: bridgeAdminLocal.address,
+      data: data,
+    });
+
+    data = localOFT.interface.encodeFunctionData("setMaxSingleTransactionLimit", [
+      remoteChainId,
+      singleTransactionLimit,
+    ]);
+    await deployer.sendTransaction({
+      to: bridgeAdminLocal.address,
+      data: data,
+    });
+
+    data = localOFT.interface.encodeFunctionData("setMaxDailyReceiveLimit", [remoteChainId, maxDailyTransactionLimit]);
+    await deployer.sendTransaction({
+      to: bridgeAdminLocal.address,
+      data: data,
+    });
+    data = localOFT.interface.encodeFunctionData("setMaxSingleReceiveTransactionLimit", [
+      remoteChainId,
+      singleTransactionLimit,
+    ]);
+    await deployer.sendTransaction({
+      to: bridgeAdminLocal.address,
+      data: data,
+    });
+
+    // Setting remote chain
+
+    await bridgeAdminRemote.setTrustedRemoteAddress(localChainId, localOFT.address);
+
+    data = remoteOFT.interface.encodeFunctionData("setMinDstGas", [localChainId, 0, 200000]);
+    await deployer.sendTransaction({
+      to: bridgeAdminRemote.address,
+      data: data,
+    });
+
+    data = remoteOFT.interface.encodeFunctionData("setMaxDailyLimit", [localChainId, maxDailyTransactionLimit]);
+    await deployer.sendTransaction({
+      to: bridgeAdminRemote.address,
+      data: data,
+    });
+    data = remoteOFT.interface.encodeFunctionData("setMaxSingleTransactionLimit", [
+      localChainId,
+      singleTransactionLimit,
+    ]);
+    await deployer.sendTransaction({
+      to: bridgeAdminRemote.address,
+      data: data,
+    });
+
+    data = remoteOFT.interface.encodeFunctionData("setMaxDailyReceiveLimit", [localChainId, maxDailyTransactionLimit]);
+    await deployer.sendTransaction({
+      to: bridgeAdminRemote.address,
+      data: data,
+    });
+    data = remoteOFT.interface.encodeFunctionData("setMaxSingleReceiveTransactionLimit", [
+      localChainId,
+      singleTransactionLimit,
+    ]);
+    await deployer.sendTransaction({
+      to: bridgeAdminRemote.address,
+      data: data,
+    });
+  });
+
+  it("send tokens from proxy oft and receive them back", async function () {
+    initialAmount = ethers.utils.parseEther("1.0000000001", 18); // 1 ether
```

Comment:
These variables can be scoped to tests rather than the describe. While multiple tests will use a value representing amounts, they aren't related to other tests and tests will likely use their own amounts.

Reply:
[15bd3cb](https://github.com/VenusProtocol/token-bridge/pull/1/commits/15bd3cbac794fb891efbde9655e85692458181a5)